spec/Section 2 -- Language.md

```diff
@@ -151,7 +151,8 @@ Token ::
   - StringValue
 
 A GraphQL document is comprised of several kinds of indivisible lexical tokens
-defined here in a lexical grammar by patterns of source Unicode characters.
+defined here in a lexical grammar by patterns of source Unicode characters. 
+Lexical tokens must be separated by either white space, line terminators, or commas.
```

Comment:
@JoeUX We can make this requirement since `[1]` is totally normal sequence of 3 tokens without any separator.

Reply:
Do you mean _can not_ ?

Reply:
@sungam3r Yes, thanks for the clarification.